### ISSUE TITLE
"Known variants" plots broken for 2020

### ISSUE THREAD

**Issue by tehwalris** [GenSpectrum/cov-spectrum-website#98]
When setting the time filter to 2020, the small plots on the "Known variants" buttons don't show any data ("0.0%"). I guess this is because we are filtering by 2020, but also by the last 3 or so months. Either we can make these plots work correctly (fetch the last 3 months of 2020), or just hide them properly (don't show "0.0%").

**Comment by ningxie1991** [comment]
Link with #106 and #126

**Comment by chaoran-chen** [comment]
Also #223

**Comment by ningxie1991** [comment]
@chaoran-chen 
I find the logic of loading the known variant list plots a bit confusing. We are getting the variants and their samples for the last 3 months (line 47). But when converting to chart data, we are selecting only the last 2 months (line 81) and also calculating the recent proportion for the last 14 days (line 109). None of the information of this 3-month, 2-month and 14-day selections is made known to the user on the website while the date range of "All Times", "2020", etc are being selected. I wonder what's the reason behind these time ranges, and shall we just make everything filter by the date range selected on the website instead?

https://github.com/cevo-public/cov-spectrum-website/blob/f8cf527afb3d40af93fa602b5f488bdcdf1b8985/src/components/KnownVariantsList/load-data.ts#L23-L121


This is what the website would look like if we set date range to 2020, get all the variants and samples between '2020-01-01' and '2020-12-31', and select the last 12 months when converting to chart data and calculate the recent proportion for last 14 days of the date range. 

![image](https://user-images.githubusercontent.com/3387698/131499073-12049e88-98aa-4545-8c11-463cc04b80e0.png)

This is what the website would look like if we set date range to 2021, get all the variants and samples between '2021-01-01' and '2021-12-31', and select the last 12 months when converting to chart data and calculate the recent proportion for last 14 days of the date range.

![image](https://user-images.githubusercontent.com/3387698/131501477-92f092a0-22c8-480e-8f0a-d740477b8eec.png)


This is how the website would look like if we set date range to "All Times" and get known variants from all time:

![image](https://user-images.githubusercontent.com/3387698/131501125-732729a5-8912-4ab5-b9e7-1785b3700434.png)
